modules/gapi/include/opencv2/gapi/cpu/gcpukernel.hpp

```diff
@@ -98,6 +106,7 @@ class GAPI_EXPORTS GCPUContext
     std::unordered_map<std::size_t, GRunArgP> m_results;
 
     friend class gimpl::GCPUExecutable;
+    friend class gimpl::render::ocv::GRenderExecutable;
```

Comment:
Is it still required here? Just because GCPUContext is reused in the implementation?

Reply:
Yes